packages/cache-utils/src/list.js

```diff
@@ -17,7 +17,7 @@ const list = async function({ cacheDir, cwd: cwdOpt, mode } = {}) {
 
 // TODO: the returned paths are missing the Windows drive
 const listBase = async function({ name, base }, cacheDir) {
-  const files = await readdirp.promise(`${cacheDir}/${name}`, { fileFilter })
+  const files = await readdirp.promise(`${cacheDir}/${name}`, { fileFilter, depth: 1, type: 'all' })
```

Comment:
Do we want `type: 'files_directories'` instead of `all`?
I would say we should at least provide an option to change `depth` and document somewhere the default.

Reply:
Good points!
Fixed in c398fbf and 5fa81bf.